src/darwin/Framework/CHIP/MTRCommissioningOperation_Internal.h

```diff
@@ -47,6 +48,11 @@ NS_ASSUME_NONNULL_BEGIN
 // Matter queue.
 @property (nonatomic, readwrite, assign) BOOL isWaitingAfterPASEEstablished;
 
+// payloadWithSuccessfulPASE, if not nil, is the payload (from possibly
+// multiple payloads represented by our self.setupPayload) that represents
+// the device we successfully established PASE with.
+@property (nonatomic, readonly, strong, nullable) MTRSetupPayload * payloadWithSuccessfulPASE;
```

Comment:
Maybe we could find a nicer name for this... what about something like "matchedPayload", "activePayload", "utilizedPayload" ?

Reply:
Not having a good name is why it wasn't public API yet.